### ISSUE TITLE
Error: missing ) in parenthetical in nightly 1.37.0 - 1.37.3+

### ISSUE THREAD

**Issue by asRIA** [kripken/emscripten#4961]
Hi,
In nightly builds, when a game is compiled in release mode, and asm.js is extracted we see an error:
`Error: missing ) in parenthetical`. Basically it's missing `)` before last `;` if asm.js file. 
I tested it in the latest sdk-nightly-1.37.3-2017_02_19_12_33-64bit and problem still exists. 
It's related to https://github.com/kripken/emscripten/issues/4836 because https://github.com/kripken/emscripten/tree/master/tools/eliminator/node_modules/uglify-js contains few local changes.
In CI when we use nightly, we based our solution on pulling uglify-js@1.0.7. That means there is no easy way to recover missing node_modules. Only one solution will be to have a local copy of correct files.

**Comment by kripken** [comment]
That's a very strange error, never seen it. Does it happen consistently? If not maybe it was a hardware error on the drive?

If it's consistent, do you have a testcase to reproduce it with?

**Comment by asRIA** [comment]
This is a ripple effect of our changes to reconstruct SDK after stripping off node_modules. We used uglyfy-js from npm, version 1.0.7 (what is checked into the repository), with this assumption we run our samples, and get an error above. 
In order to fix it, we had to brutally clone https://github.com/kripken/emscripten/tree/master/tools/eliminator/node_modules/ to our local repo and use it. 

This issue is more like a manifest about importance of not removing node_modules unless a safe alternative is given.
